test/fixtures/mutating_webhook_configurations/secret_hook.yaml

```diff
@@ -18,7 +18,7 @@ webhooks:
   reinvocationPolicy: Never
   rules:
   - apiGroups:
-    - core
+    - ""
```

Comment:
Are these actually equivalent? Does `""` imply `core`. What broke that required this change?

Reply:
I would need to create a copy of the Krane class specifically for this test. Since Secret.core isn't a valid resource anymore. This doesn't break for current master since it doesn't care about group anyways.